.gitignore

```diff
@@ -1,3 +1,7 @@
+*
+!**/
+!*.*
+
```

Comment:
I hope you don't mind adding these "standard" Nim gitignores which allow to keep in-place binaries (files without a dot) without seeing untracked files.

Reply:
usually I just put everything in my build/ folder